.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 
-sudo: false
```

Comment:
Hi all, I think the proper way is adding the ```sudo: false``` for each time so that it can always make sure that it's running on the non-root  the environment.

What do we think?

Thanks.